openreview/openreview.py

```diff
@@ -419,26 +428,26 @@ def post(self, client):
         client.post_group(self)
 
 class Invitation(object):
-    def __init__(self, id, writers=None, invitees=None, noninvitees=None, readers=None, nonreaders=None, reply=None, web=None, process=None, signatures=None, duedate=None, cdate=None, rdate=None, ddate=None, multiReply=None, taskCompletionCount=None):
+    def __init__(self, id, writers=None, invitees=None, noninvitees=None, readers=None, nonreaders=None, reply=None, replyto=None, forum=None, web=None, process=None, signatures=None, duedate=None, cdate=None, rdate=None, ddate=None, multiReply=None, taskCompletionCount=None):
 
         default_reply = {
-            'forum': None,
-            'replyto': None,
-            'readers': {'values': ['everyone']},
-            'signatures': {'values-regex': '.*'},
-            'writers': {'values-regex': '.*'},
-            'content':{}
-        },
+            'forum': forum,
+            'replyto': replyto,
+            'readers': {},
+            'signatures': {},
+            'writers': {},
+            'content': {}
+        }
```

Comment:
I took out the defaults here, but I left the "default_reply" dictionary, because I think it's helpful to see which fields are expected. What do you think @melisabok ?